src/mentioncommand.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module mention/mentioncommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+import toMap from '@ckeditor/ckeditor5-utils/src/tomap';
+import uid from '@ckeditor/ckeditor5-utils/src/uid';
+
+/**
+ * The mention command.
+ *
+ * The command is registered by the {@link module:mention/mentionediting~MentionEditing} as `'mention'`.
+ *
+ * To insert mention on range, execute the command and specify, mention object and range to replace:
+ *
+ *		const focus = editor.model.document.selection.focus;
+ *
+ *		editor.execute( 'mention', {
+ *			mention: {
+ *				name: 'Foo',
+ *				id: '1234',
+ *				title: 'Big Foo'
+ *			},
+ *			marker: '#',
+ *			range: model.createRange( focus, focus.getShiftedBy( -1 ) )
+ *		} );
+ *
+ * @extends module:core/command~Command
+ */
+export default class MentionCommand extends Command {
+	/**
+	 * @inheritDoc
+	 */
+	refresh() {
+		const model = this.editor.model;
+		const doc = model.document;
+
+		this.isEnabled = model.schema.checkAttributeInSelection( doc.selection, 'mention' );
+	}
+
+	/**
+	 * Executes the command.
+	 *
+	 * @param {Object} [options] Options for the executed command.
+	 * @param {Object|String} options.mention Mention object to insert. If passed a string it will be used to create a plain object with
```

Comment:
```suggestion
	 * @param {Object|String} options.mention Mention object to insert. If a string is passed, it will be used to create a plain object with
```